### PR TITLE
feat: add issues app

### PR DESCRIPTION
**Pull request by satyarohith**
The deploy instructions will be updated later.﻿

### CODE REVIEW

File: issues/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Run Offline
+
+You can run the application on your local machine using `denodeploy`.
+
+```
+GITHUB_TOKEN=<token> denodeploy https://raw.githubusercontent.com/denoland/deploy_examples/issues/mod.js
+```
+
+Create a personal ccess token at https://github.com/settings/tokens without any
```
Comment (kt3k) [inline issues/readme.md:25]:
```suggestion
Create a personal access token at https://github.com/settings/tokens without any
```

File: issues/pages/api/issues.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  if (!repository) {
+    repository = repositories[Math.floor(Math.random() * 500)];
+  }
+
+  // Retrieve GitHub API token from env. Error if not set.
+  const token = Deno.env.get("GITHUB_TOKEN");
+  if (!token) {
+    return {
+      code: "tokenNotAvailable",
+      message: "Environment variable GITHUB_TOKEN not set.",
```
Comment (kt3k) [inline issues/pages/api/issues.js:54]: This error message was very helpful when deploying this app to my sr project 👍

**Review by kt3k — APPROVED** [review]
Great example!

**Comment by kt3k** [comment]
I saw some errors when loading `nodejs/node` or `microsoft/TypeScript`. Might have some problem with too large responses?

スクリーンショット 2021-03-23 12 36 29

スクリーンショット 2021-03-23 12 42 09

I was able to load `denoland/deno`, `babel/babel`, `facebook/react`.

スクリーンショット 2021-03-23 12 28 35

**Comment by satyarohith** [comment]
I will take a look.

**Comment by satyarohith** [comment]
Turns out GitHub throws a "Server Error" for a large enough repository on the first request but succeeds on the following request. It's addressed in the new commit.

**Pull request merged**